package.json

```diff
@@ -41,7 +41,7 @@
     "@storybook/components": "^5.2.5",
     "copy-to-clipboard": "^3.0.8",
     "js-beautify": "^1.8.8",
-    "react-element-to-jsx-string": "^14.0.3"
+    "react-element-to-jsx-string": "14.3.1"
```

Comment:
What's the reason for making this version exact?

Reply:
That would be my `~/.npmrc` with `save-exact=true`. But I didn't think about it when I did it. 

~~"^14.3.1" should work, as long as it doesn't allow for "14.3.0" to be used.~~ I tried it and it doesn't: 

```js
var semver = require('semver')
semver.satisfies('14.3.0', '^14.3.1') // true
```

I'll change it back to match the style of the project.